### ISSUE TITLE
Wrapper new Bitwise And Or XOR (Not Logic XOR) in tensorflow

### ISSUE THREAD

**Issue by yylonly** [keras-team/keras#7090]
Tensorflow just adds a bitwise module with bitwise_and, bitwise_or, bitwise_xor, and i…

in this PR https://github.com/tensorflow/tensorflow/commit/a94a333762b4d2c3fcae0a9e610d654bba6980de

how could I wrapped the new Tensorfolw in Keras, then I could build new layer using K.bitwiseXor()

Thanks.

I know Keras have bitwise Op but that is LogicAND OR XOR but Bitwise AND OR XOR.... Thanks.

if there have some tutorials, I can contribute this new wrapper.

**Comment by Dref360** [comment]
There is no gradient defined for theses ops so they cannot be used for learning. To use them in metrics, you can use a Lambda layer.

**Comment by yylonly** [comment]
Lambda layer is a good idea!  I have no parameters in my self-defined layers, that is no need to define derivation of the gradient.

**Comment by waleedahmedrana** [comment]
@yylonly Hello
I also need bitwise xor in a non trainable layer. The use of lambda could also work. Have you been able to get it to work? Could you please share your work?

Thanks.
